back/spec/services/url_validation_service_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
+
+describe UrlValidationService do
```

Comment:
This file should also test `video_whitelisted?`, so:
```
describe '#video_whitelisted?' do
  ...
end
```

Reply:
Ah true - I will add in those test.

Reply:
@koendehondt Should I remove the tests in sanitization_service_spec.rb then that test the video whitelist from there?

Reply:
No, because they are concerned with "sanitization".